### DISCUSSION TITLE
Login with github without --link or 2FA

### DISCUSSION THREAD

**Discussion by TheScientist101** [coder/code-server#2905] [Ideas] [answered]
Is it possible to log in with GitHub instead of a password(without --link) or some sort of 2FA, I think that this would just make it feel safer. Preferably the GitHub Login.

**Comment by code-asher** [comment, marked as answer]
At the moment we've decided not to implement any other kinds of authentication. Our current recommendation is to use an oauth proxy although I think we need to rework the docs a bit to make that more obvious.

Related issue: https://github.com/cdr/code-server/issues/2636
